core-blocks/code/index.js

```diff
@@ -4,13 +4,12 @@
 import { __ } from '@wordpress/i18n';
 import {
 	createBlock,
-	PlainText,
 } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import './editor.scss';
+import { edit } from './edit'; // import the specialized `edit` function. Is is different between the web and native mobile.
```

Comment:
This comment can be removed as we should probably do the same for all `edit` implementations :)

`edit` can be exported as default to have:

```js
import edit from './edit';

// and later
{ 
     ...
    edit,
    ...
}
```

Reply:
The comment can be considered trivial if one reads the PR and has an understanding of the `.native.js` pattern. I had added it to help people pick up the pattern but, i'm perfectly fine with removing it 👍 . Done with 9b68ba3.